package.json

```diff
@@ -14,19 +14,14 @@
     "web": "expo start --web",
     "prepare": "husky install",
     "check:writing": "alex .",
-    "check:types": "tsc --noemit",
```

Comment:
Removed this script and stopped using the `tsc` compiler because there are "duplicate identifier" errors everytime the typescript compiler is ran. The issue seems to be related to `react-native` and `typescript` using the same names for globally accessible types (in .d.ts files). I tried a few things including `skipLibCheck: true` in our tsconfig but it didn't help. Since we are not using `tsc` to actually generate any js code I think it's ok, not 100% sure though. Wdyt @sandrina-p?

Reply:
If you want to see what type of errors I'm talking about you can run `yarn tsc src/app/index.tsx`

Reply:
So the issue is that if you run `tsc` on a specific file it ignores the `tsconfig`. I found [tsc-files](https://www.npmjs.com/package/tsc-files) that does exactly want I wanted and fixed the issues I was having. Now we can properly run `tsc` on every typescript file we commit 😄

Reply:
I'm far from an expert in TS, so whatever makes our DX easier, I'm fine! If you think this is the best, I trust you ✌️

Reply:
The stuff that I talked about here was addressed in https://github.com/cratera/papers-game/pull/26 btw, I parked the nativewind stuff for now

Reply:
Okay, so you could prefix this MR with "Draft:" or "WIP" to know that it's on hold for now. (it's a common practice from gitlab)

Reply:
Done, but interesting that you were even prompted to review it since it was already a draft PR 🤔

Reply:
Ohhhh I realized what was wrong a minute ago. I thought it was ready because I got notified. Only now I realize that my default settings was to be notified in every change to this repo (new PR and commits to those PRs). No worry, it's all disabled now, pls cc me when it's ready :D